packages/stats-py/aqsha_stats/analyses/cb_sem.py

```
@@ -0,0 +1,195 @@
+RMSEA_CUTOFF = 0.08
+
+
+def _num(value) -> float | None:
+    try:
+        f = float(value)
+    except (TypeError, ValueError):
+        return None
+    return f
+
+
+def run(df: pd.DataFrame, args: dict) -> dict:
```

Ruff (0.15.20)

[warning] Too many branches (13 > 12)

(PLR0912)